src/battle/battle_setup.c

```diff
@@ -466,29 +466,29 @@ void sub_02052444(BattleSetup* setup, FieldSystem* fieldSystem) {
 }
 
 static const u32 _020FC4C0[] = {
```

Comment:
any chance to name this var, smth like BattleBgTerrainMap?

Reply:
Let's keep the renaming of variables and functions to its own PR.

Reply:
I mean, it's not massive but if you want to do it separately then sure